File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] a = {0, 0};
        for(int i = 0; i < numbers.length; i++){
            for(int j = 0; j < numbers.length; j++){
                if(i != j && numbers[i] + numbers[j] == target){
                    a[0] = i+1;
                    a[1] = j+1;
                    return a;
                }
            }
        }
        return a;
    }
}

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-0.cpp
class Solution {
public:
    int minSubArrayLen(int target, vector<int>& nums) {
        int mn = INT_MAX;
        bool ok = 0;
        for(int i = 0; i < nums.size(); i++){
            int sum = 0;
            int k = 0;
            for(int j = i; j < nums.size(); j++){
                sum += nums[j];
                k++;
                if(sum >= target){
                    mn = min(mn, k);     
                    ok = 1;  
                }
            }
        }
        if(!ok) return 0;
        return mn;
    }
};